RULES.md

```diff
@@ -59,6 +59,14 @@ An email should be a valid email address (e.g., contact@agency.org)
 #### References:
 * [Field Types Description](http://gtfs.org/reference/static/#field-types)
 
+### E024 - Route short name too long
```

Comment:
Update this file with rule E024, however I think that this should be a warning instead of an error since it's described by the GTFS best practices and not the GTFS static specification. @maximearmstrong, @fabrice-v ?